### DISCUSSION TITLE
NetAlertX keep on "Loading" while top right showing "Import config"

### DISCUSSION THREAD

**Discussion by chengidesk** [netalertx/NetAlertX#863] [Q&A]
Hi @jokob-sk, 

I wasn't able to touch on anything from likely early of the month or late last month until today but I noticed there is a power failure in between and now NetAlertX is not loading. Any idea what can I do to resolve this?
![image](https://github.com/user-attachments/assets/3a37632d-7d8c-40b1-8f03-f45578ac43f1)
![image](https://github.com/user-attachments/assets/35298b82-71e9-4499-96a3-f2324dd7c28c)

thank you.

Regards,

**Comment by jokob-sk** [comment]
Hi @chengidesk ,

It looks like again that the DB grew too big. I would recommend looking into  the steps in the performance docs to make it smaller: 

https://github.com/jokob-sk/NetAlertX/blob/main/docs/PERFORMANCE.md

It also seems that the app is restarting for an unknown reason. Please share your logs from docker or Portainer directly as the in-app log can't capture exceptions.

**Reply by chengidesk** [reply]
unfortunately, I re-install it earlier but here's the log,
[_suspicious_lamarr_logs.txt](https://github.com/user-attachments/files/17505893/_suspicious_lamarr_logs.txt)

**Reply by jokob-sk** [reply]
Thank you - it seems like the config file may have been damaged during the outage. The following line should contain a value:

```
PLUGINS_KEEP_HIST=
```

for example:

```
PLUGINS_KEEP_HIST=5000
```

You can try to edit or restore the `app.conf` file, but there might be multiple values missing.

**Reply by chengidesk** [reply]
Thanks @jokob-sk, I just add back the "5000" now it is working but it is showing significant of devices online while there should not be anyone at school now.

**Reply by jokob-sk** [reply]
Sorry, without logs I can't help. I would also recommend waiting a few minutes and clearing the cache.

**Reply by chengidesk** [reply]
The CPU and Memory spike again. This time I increased to 10 core and 8GB RAM and I've disabled all scanning. 
[app.log](https://github.com/user-attachments/files/17506855/app.log)

After I click run on both DB Cleanup and Maintenance, the storage size still at 490,09MB
![image](https://github.com/user-attachments/assets/26084ad7-45af-48b1-836c-9d237ebdfb6b)

**Reply by jokob-sk** [reply]
After running the DB cleanup plugin, did these numbers go down? 

![image](https://github.com/user-attachments/assets/fb060944-1a01-42d3-b546-fc1b73e1ecd7)

If not, the task timed out and I would recommend to increase the timeout. If the numbers still don't go down, please share the log section when you run the DB cleanup plugin.
